lib/matplotlib/pyplot.py

```diff
@@ -307,58 +307,103 @@ def draw_if_interactive(*args, **kwargs):
 # This function's signature is rewritten upon backend-load by switch_backend.
 def show(*args, **kwargs):
     """
-    Display all figures.
+    Display all open figures.
 
-    When running in ipython with its pylab mode, display all
-    figures and return to the ipython prompt.
+    In non-interactive mode, **block** defaults to True.  All figures
```

Comment:
Single `*` for parameters.
```suggestion
    In non-interactive mode, *block* defaults to True.  All figures
```